frontend/src/components/App.css

```
@@ -1,123 +1,769 @@
+  font-size: 13.5px;
+  color: var(--text-primary);
+  border-bottom: 1px solid var(--border);
+}
+
+.fairway-table tr:last-child td { border-bottom: none; }
+.fairway-table tr:hover td { background: var(--primary-faint); }
+
+.td-course-name { font-weight: 500; }
+.td-location { color: var(--text-secondary); font-size: 13px; }
+.td-year {
+  font-family: 'Inter', monospace;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)